thrift/thrift0.19/src/test/java/com/linecorp/armeria/common/thrift/logging/it/ThriftContentSanitizerITTest.java

```diff
@@ -0,0 +1,96 @@
+import static org.awaitility.Awaitility.await;
+
+import java.util.Map;
+
+import org.apache.thrift.TException;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.RegisterExtension;
+
+import com.linecorp.armeria.client.thrift.ThriftClients;
+import com.linecorp.armeria.common.HttpResponse;
+import com.linecorp.armeria.common.logging.ContentSanitizer;
+import com.linecorp.armeria.common.logging.FieldMasker;
+import com.linecorp.armeria.common.thrift.logging.ThriftFieldMaskerSelector;
+import com.linecorp.armeria.server.ServerBuilder;
+import com.linecorp.armeria.server.logging.LoggingService;
+import com.linecorp.armeria.server.thrift.THttpService;
+import com.linecorp.armeria.testing.junit5.server.ServerExtension;
+
+import testing.thrift.main.SecretService;
+import testing.thrift.main.SecretStruct;
+
+class ThriftContentSanitizerITTest {
+
+    @RegisterExtension
+    static ServerExtension server = new ServerExtension() {
+        @Override
+        protected void configure(ServerBuilder sb) throws Exception {
+            sb.service("/", (ctx, req) -> HttpResponse.of(200));
+        }
+    };
+
+    @Test
+    void basicCase() throws Exception {
+        final ContentSanitizer<String> contentSanitizer = commonContentSanitizer();
+        final TestLogWriter logWriter = new TestLogWriter(contentSanitizer);
+
+        server.server().reconfigure(sb -> {
+            sb.decorator(LoggingService.builder().logWriter(logWriter).newDecorator());
+            sb.service("/", THttpService.builder()
+                                        .addService(new SecretService.Iface() {
+                                            @Override
+                                            public SecretStruct hello(SecretStruct req) throws TException {
+                                                return req;
+                                            }
+                                        })
+                                        .build());
+        });
+
+        final SecretService.Iface iface = ThriftClients.newClient(server.httpUri(), SecretService.Iface.class);
+        final SecretStruct req = new SecretStruct().setSecret("secret").setHello("hello");
+        final SecretStruct res = iface.hello(req);
+        assertThat(res).isEqualTo(req);
+
+        await().untilAsserted(() -> assertThat(logWriter.blockingDeque()).hasSize(2));
+
+        final String requestLog = logWriter.blockingDeque().takeFirst();
+        assertThat(requestLog).contains("hello");
+        assertThat(requestLog).doesNotContain("secret");
+
+        final String responseLog = logWriter.blockingDeque().takeFirst();
+        assertThat(responseLog).contains("hello");
+        assertThat(requestLog).doesNotContain("secret");
+    }
+
+    private static ContentSanitizer<String> commonContentSanitizer() {
+        return ContentSanitizer.builder()
+                               .fieldMaskerSelector(ThriftFieldMaskerSelector.of(info -> {
```

Comment:
What do you think of adding a builder to create a `ThriftFieldMaskerSelector` in a declarative way for simple cases?

For example:
```java
ThriftFieldMaskerSelector
   .buider()
   .onFieldAnnotation("secret", FieldMasker.nullify())
   .onFieldAnnotation("grade", "red", new CustomFieldMasker())
   ...
   .build();
```

Reply:
It looks nice. Should we add a integration test case using `ThriftFieldMaskerSelectorBuilder` as an example so that users can reference it?